src/styles/index.css

```diff
@@ -169,6 +169,7 @@ img {
 }
 
 .drop-container img {
+  -webkit-user-drag: none;
```

Comment:
This works only for Chrome and Safari (which is already nice) - If you put draggable="false" as image-property it works also e.g. in FF. Or maybe even better - if we just don't add an image to the dropContainer.